### PR TITLE
Philip/local focal

### PR DESCRIPTION
**Pull request by philipkiely-baseten**

### CODE REVIEW

File: docs/create/huggingface.md [hunk 1]
```diff
@@ -31,7 +31,17 @@ Use the `mk_truss` command to package your model into a Truss.
...
 ```
 
 Check the target directory to see your new Truss!
+
+### Serve the model
+
+To get a prediction from the Truss, try running:
+
+```python
+tr.docker_predict({"inputs": "TODO"})
```
Comment (howespt) [inline docs/create/huggingface.md:44]: Is this input a TODO or are you trying to complete a sentence starting with "TODO"

File: docs/create/huggingface.md [hunk 2]
```diff
@@ -31,7 +31,17 @@ Use the `mk_truss` command to package your model into a Truss.
...
 ```
 
 Check the target directory to see your new Truss!
+
+### Serve the model
+
+To get a prediction from the Truss, try running:
+
+```python
+tr.docker_predict({"inputs": ["Donatello is a teenage mutant [MASK] turtle"]})
```
Comment (howespt) [inline docs/create/huggingface.md:44]: he sure is

File: docs/notebooks/huggingface_example.ipynb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Truss TensorFlow example\n",
+        "\n",
+        "<a href=\"https://colab.research.google.com/github/basetenlabs/truss/blob/main/docs/notebooks/sklearn_example.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (philipkiely-baseten) [inline docs/notebooks/huggingface_example.ipynb:23]: fix url

File: docs/create/manual.md [hunk 4]
```diff
@@ -2,12 +2,22 @@
...
 ```
 truss init my_truss
 ```
 
+From there, as pseudocode, you'll do the following:
+
+```python
+my_model = train_model()
+my_model.serialize(save_to="path/to/my_truss")
+tr = truss.load("path/to/my_truss")
```
Comment (pankajroark) [inline docs/create/manual.md:16]: This code doesn't look right.

File: docs/create/manual.md [hunk 5]
```diff
@@ -36,14 +46,54 @@ The model file implements the following functions, in order of execution:
...
+```
+truss init my_truss
+```
+
+This is the part you want to replace with your own code. Build a machine learning model and keep it in-memory.
+
+```python
+import xgboost as xgb
+from sklearn.datasets import make_classification
+from sklearn.model_selection import train_test_split
```
Comment (pankajroark) [inline docs/create/manual.md:72]: nit: newline here for readability.

File: docs/create/manual.md [hunk 6]
```diff
@@ -36,14 +46,54 @@ The model file implements the following functions, in order of execution:
...
+def create_data():
+    X, y = make_classification(n_samples=100,
+                           n_informative=5,
```
Comment (pankajroark) [inline docs/create/manual.md:75]: indentation

File: docs/create/manual.md [hunk 7]
```diff
@@ -36,14 +46,54 @@ The model file implements the following functions, in order of execution:
...
+    train = xgb.DMatrix(X_train, y_train)
+    test = xgb.DMatrix(X_test, y_test)
+    return train, test
+train, test = create_data()
+params = {
+    "learning_rate": 0.01,
+    "max_depth": 3
+}
+# training, we set the early stopping rounds parameter
+model = xgb.train(params,
```
Comment (pankajroark) [inline docs/create/manual.md:87]: very minor. Indentation doesn't look right. Folks will copy-paste, best if they don't get warnings doing so.

File: docs/create/pytorch.md [hunk 8]
```diff
@@ -144,7 +144,17 @@ Use the `mk_truss` command to package your model into a Truss.
...
 ```
 
 Check the target directory to see your new Truss!
+
+### Serve the model
+
+To get a prediction from the Truss, try running:
+
+```python
+tr.docker_predict({"inputs": "TODO"})
```
Comment (pankajroark) [inline docs/create/pytorch.md:157]: is the TODO intentional?

**Review by pankajroark — APPROVED** [review]
minor comments. Fixing the example code seems important.

**Pull request merged**